### DISCUSSION TITLE
Remove shadow from text

### DISCUSSION THREAD

**Discussion by edualvarado** [mmistakes/minimal-mistakes#2437] [Q&A] [answered]
Hello,

I have a "splash" page with some text. Additionally, I am using a white background. The text, however, shows a shadow that I would like to remove, to leave a normal font. 

This is the current appearance: [https://i.imgur.com/T5HDDe8.jpg](url)

Is it possible to remove such shadow? Thanks!

**Comment by iBug** [comment, marked as answer]
Add `box-shadow: none;` to the text and the shadow is gone.

**Comment by edualvarado** [comment]
Perfect, thank you! It worked.
